cmd/bundle/generate/app.go

```diff
@@ -36,8 +41,8 @@ func NewGenerateAppCommand() *cobra.Command {
 	cmd.Flags().StringVar(&appName, "existing-app-name", "", `App name to generate config for`)
 	cmd.MarkFlagRequired("existing-app-name")
 
-	cmd.Flags().StringVarP(&configDir, "config-dir", "d", filepath.Join("resources"), `Directory path where the output bundle config will be stored`)
-	cmd.Flags().StringVarP(&sourceDir, "source-dir", "s", filepath.Join("src", "app"), `Directory path where the app files will be stored`)
+	cmd.Flags().StringVarP(&configDir, "config-dir", "d", appDefaultConfigDir, `Directory path where the output bundle config will be stored`)
```

Comment:
getting rid of filepath.Join is good, but why split the definition?

Since the constant is not used anywhere else, inline defaults were easier to read.

Reply:
That's fair, removed them.